hashintel/hash
# Add `createOrg` mutation

## 🌟 What is the purpose of this PR?




This PR introduces a `createOrg` GQL mutation, which can be used by HASH instance admins to create an organization.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203150072758509/1203249971975775/f) _(internal)_

## 🔍 What does this change?




See commits.

## 📜 Does this require a change to the docs?



Not that I'm aware of.

## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



None, the GQL API mutation was tested manually.

## ❓ How to test this?



1. Checkout the branch and run the HASH App locally
2. Go to localhost:3000 and login as a HASH instance admin
3. Access the GQL playground at `localhost:5001/graphql`
4. Run following query, and inspect the response to see the created org entity
```GraphQL
mutation createOrg {
  createOrg(
    shortname: "test-org",
    name: "test",
    orgSize: ELEVEN_TO_FIFTY
  ) {
    roots
  }
}
```
